Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -163,6 +172,30 @@ public extension UIConfig {
             MessageActionsView<ExtraData>.self
         public var actionButton: MessageActionsView<ExtraData>.ActionButton.Type =
             MessageActionsView<ExtraData>.ActionButton.self
+        public var actionIcons: [ChatMessageActionItem.Name: UIImage] = [
+            .inlineReply: UIImage(named: "icn_inline_reply", in: .streamChatUI)!,
+            .threadReply: UIImage(named: "icn_thread_reply", in: .streamChatUI)!,
+            .edit: UIImage(named: "icn_edit", in: .streamChatUI)!,
+            .copy: UIImage(named: "icn_copy", in: .streamChatUI)!,
+            .unblockUser: UIImage(named: "icn_block_user", in: .streamChatUI)!,
+            .blockUser: UIImage(named: "icn_block_user", in: .streamChatUI)!,
+            .muteUser: UIImage(named: "icn_mute_user", in: .streamChatUI)!,
+            .unmuteUser: UIImage(named: "icn_mute_user", in: .streamChatUI)!,
+            .delete: UIImage(named: "icn_delete", in: .streamChatUI)!,
+            .resend: UIImage(named: "icn_resend", in: .streamChatUI)!
+        ]
+        public var actionTitles: [ChatMessageActionItem.Name: String] = [
+            .inlineReply: L10n.Message.Actions.inlineReply,
+            .threadReply: L10n.Message.Actions.threadReply,
+            .edit: L10n.Message.Actions.edit,
+            .copy: L10n.Message.Actions.copy,
+            .unblockUser: L10n.Message.Actions.userUnblock,
+            .blockUser: L10n.Message.Actions.userBlock,
+            .muteUser: L10n.Message.Actions.userMute,
+            .unmuteUser: L10n.Message.Actions.userUnmute,
+            .delete: L10n.Message.Actions.delete,
+            .resend: L10n.Message.Actions.resend
```

Comment:
Is there any reason why user would like to override default titles for message actions?

Reply:
after the internal discussing decided that titles shouldn't be customizable -> removed from `UIConfig`